### PR TITLE
networking: Don't use checkpoints for bridges, bonds, etc.

### PR DESCRIPTION
**Pull request by mvollmer**
We would run into known bugs and the results are too confusing.

See https://bugzilla.redhat.com/show_bug.cgi?id=1395108

- [x] #5472

### CODE REVIEW

File: pkg/networkmanager/interfaces.js [hunk 1]
```diff
@@ -1950,6 +1974,20 @@ function with_checkpoint(model, modify, fail_text, anyway_text) {
...
+    // HACK - Let's not use checkpoints for changes that involve
+    // adding or removing connections.
+    //
+    // https://bugzilla.redhat.com/show_bug.cgi?id=1378393
+    // https://bugzilla.redhat.com/show_bug.cgi?id=1398316
+    //
+    // These bugs are expected to be fixed in NM 1.6
+
+    if (hack_does_add_or_remove && !model.at_least_version("1.6")) {
+        console.log("nope");
```
Comment (stefwalter) [inline pkg/networkmanager/interfaces.js]: I think this is an extra line no?

**Comment by stefwalter** [comment]
Does this need an integration test, or an integration test adapted?

**Comment by mvollmer** [comment]
> Does this need an integration test, or an integration test adapted?

I try to test the original scenario that prompted this change: putting the main interface into a bond when DHCP is really slow.  This should eventually succeed and not cause a rollback.

File: pkg/networkmanager/interfaces.js (hunk 1 above)
Comment (mvollmer) [inline pkg/networkmanager/interfaces.js]: Fixed.

**Comment by mvollmer** [comment]
Priority because of https://bugzilla.redhat.com/show_bug.cgi?id=1395108

**Pull request closed without merging**